l10n/en-US/viewer.ftl

```diff
@@ -493,3 +493,20 @@ pdfjs-editor-alt-text-settings-editor-title = Alt text editor
 pdfjs-editor-alt-text-settings-show-dialog-button-label = Show alt text editor right away when adding an image
 pdfjs-editor-alt-text-settings-show-dialog-description = Helps you make sure all your images have alt text.
 pdfjs-editor-alt-text-settings-close-button = Close
+
+## "Annotations removed" bar
+
+pdfjs-editor-undo-bar-message-highlight = Highlight removed
+pdfjs-editor-undo-bar-message-freetext = Text removed
+pdfjs-editor-undo-bar-message-ink = Drawing removed
+pdfjs-editor-undo-bar-message-stamp = Image removed
+# Variables:
+#   $count (Number) - the number of removed annotations.
+pdfjs-editor-undo-bar-message-multiple = { $count } annotations removed
```

Comment:
```suggestion
pdfjs-editor-undo-bar-message-multiple = 
    { $count ->
        [one] { $count } annotation removed
       *[other] { $count } annotations removed
    }
```

This needs a plural form.

Reply:
@flodolo the message is actually never rendered unless the number is two or more but I imagine it'll make it easier to port it to other languages with more plural forms?

Reply:
Other languages have more [complex plural rules](https://www.unicode.org/cldr/charts/45/supplemental/language_plural_rules.html) than English (up to 6 forms). We want to instruct tools that this string needs plural support, even if the `one` category is never used for English.

Reply:
fair enough, done!